src/main/java/pl/kzcwat/localincidentserver/region/RegionController.java

```diff
@@ -0,0 +1,80 @@
+package pl.kzcwat.localincidentserver.region;
+
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+import pl.kzcwat.localincidentserver.region.exception.RegionNotFoundException;
+import pl.kzcwat.localincidentserver.region.request.RegionRequest;
+
+import java.util.Optional;
+
+@RequiredArgsConstructor
+@RestController
+@RequestMapping("/api/region")
```

Comment:
Przydałoby się wersjonowanie api. Czyli powinno być 
```suggestion
@RequestMapping("api/v1/region")
```